bootstrap-app/src/cmr/bootstrap/services/bootstrap_service.clj

```diff
@@ -26,6 +26,8 @@
    :index-collection :core-async-dispatcher
    :index-system-concepts :core-async-dispatcher
    :index-concepts-by-id :core-async-dispatcher
+   :fingerprint-by-id :synchronous-dispatcher
+   :fingerprint-variables :core-async-dispatcher
```

Comment:
Using a core-async dispatcher means we can't distribute the work across multiple nodes. Is it fine to use one node to recalculate the fingerprints for all variables in the system?

Reply:
Good point. I have switched it to be dispatch through message queue on a per provider basis.